apis/nucleus/src/components/listbox/ListBoxPortal.jsx

```diff
@@ -36,7 +37,18 @@ function ListBoxWrapper({ app, fieldIdentifier, qId, stateName, element, options
     model,
   };
 
-  return <ListBoxInline options={opts} />;
+  return opts.popover ? (
+    <ListBoxPopover
+      app={app}
+      alignTo={{ current: element }}
+      show={options.popoverState}
```

Comment:
@Caele @veinfors Do you know why we have an option here for showing/hiding the popover? It feels like an unnecessary option since the user of the Listbox can just do a conditional rendering instead:
```
popoverState && renderTheListbox()
```

Will it optimise the code by preventing running some code each time – or is it just a remain from the past?

Reply:
as you suggested, conditionally rendering listbox from our side in table works fine, but if we do so, the questions becomes what would be the use case for `show` property in `<ListBoxPopover>` component

if we render it conditionally, i need to pass an explicit `true` for that prop.

Reply:
the prop for showing the listboxpopover is for when Nebula uses it internally, such as in the selections toolbar. When we go to expose the popover to the outside, we don't want to expose the react implementation so we need some other way.

Reply:
Sounds like we should not expose it to the user and instead have a fallback to `show == true` – if that doesn't destroy that internal implementation.